source/fundamentals/builders/sort.txt

```diff
@@ -11,3 +11,346 @@ Sorts Builders
    :class: singlecol
 
 .. _sorts-builders:
+
+Overview
+--------
+
+In this guide, we show you how to use **builders** to specify
+**sort criteria** for your queries.
+
+Sort criteria are the rules MongoDB uses to sort your data. Some examples of sort criteria are:
+
+* Smallest number to largest number
+* Earliest time of day to latest time of day
+* Alphabetical order by first name 
+
+Builders are classes provided by the MongoDB Java driver that allow you to use
+the builder pattern to interact with your MongoDB instance.
+The builder pattern allows you to create complex
+immutable objects separately from their representation. The
+:doc:`builders </fundamentals/builders/>`
+provided by the MongoDB Java driver can improve the readability of your code.
+
+You should read this guide if you would like to:
+
+* Use the builder pattern to specify sort criteria for your queries.
+* Learn how to perform ascending sorts, descending sorts, and combining sorts.
+* Sort on the text score of :manual:`$text </reference/operator/query/text/>` 
+  queries.
+
+If you are new to sorting in the
+MongoDB Java driver, consider reading our
+:doc:`guide on sorting </fundamentals/crud/read-operations/sort/>`
+for a general introduction to sorting with the MongoDB Java driver.
+
+The Sorts Class
+---------------
+
+The :java-docs:`Sorts </apidocs/mongodb-driver-core/com/mongodb/client/model/Sorts.html>`
+class is the sort criteria builder provided by the MongoDB Java Driver.
+``Sorts`` provides static factory methods for all
+sort criteria operators supported by MongoDB. Each method returns
+an instance of the
+:java-docs:`Bson </apidocs/bson/org/bson/conversions/Bson.html>`
+type. 
```

Comment:
1) Nitpick: I think we can group them as "all" rather than "each" since we're identifying that they have the same behavior, not an individually different one.

2) It would be useful to have more information on what's significant about the Bson type, e.g. 
All these methods return a ``Bson`` object that you can pass to the `sort()` method.

Reply:
Thanks for the note. I am referring to the group of methods in Sorts (rather than all individual methods in Sorts), so you are right "All" is a better term. Additionally, on having it pointed out, I can see how a user would not know what to do with "returns a ``bson`` type". I am taking your suggested changes, and adding a bit of text to clarify that ``sort()`` is a method of a ``FindIterable`` instance. Fix: https://github.com/mongodb/docs-java/pull/59/commits/b8b071b5042fc8bcab62ce100233f14641b36c60